metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -233,6 +233,31 @@ def parse_restli_response(response):
     return rows
 
 
+@ingest.command()
+@click.argument("path", type=click.Path(exists=True))
+def metadata_file(path: str) -> None:
+    """
+    Ingest from a metadata json file.
+
+    This requires that you've run `datahub init` to set up your config.
```

Comment:
It shouldn't require it since sink is auto-inferred.

Reply:
The sink is inferred from your env variables or your ~/.datahubenv config - so they do need to run init first